docs/operation/cluster_and_service_configuration.rst

```diff
@@ -324,10 +324,41 @@ This functionality can be actived:
    than `retention_size`. You should at least add a 10% margin to be safe.
    (i.e.: set `retention_size` to 9GB for a 10GB volume)
 
-Both size and time based retentions can be activated at the same time.
+Then :ref:`apply the configuration<csc-prometheus-apply-cfg>`.
+
+Set Kubelet metrics scrape tiemout
+""""""""""""""""""""""""""""""""""
+
+When using a bunch of sparseloopdevice kubelet metrics endpoint may spend
+a lot of time to retrieve volumes metrics. In such situation the default
+10s scrape timeout of Prometheus is not enough as the metrics endpoint may
+take longer to answer.
+
+To avoid losing data in such situations, we enabled customization of kubelet
+scarpe timeout, to customize it use the following configuration:
+
+
+.. code-block:: yaml
+
+   ---
+   apiVersion: v1
+   kind: ConfigMap
+   metadata:
+     name: metalk8s-prometheus-config
+     namespace: metalk8s-monitoring
+   data:
+     config.yaml: |-
+       apiVersion: addons.metalk8s.scality.com
+       kind: PrometheusConfig
+       spec:
+         config:
+          kubelet_scrape_timeout: 30s
 
 Then :ref:`apply the configuration<csc-prometheus-apply-cfg>`.
 
+Kubelet metrics scrape tiemout and both size and time based retentions can be
+activated at the same time.
```

Comment:
For me this information is only relevant for the retention, because it could be exclusive, you should let it where it was.

Reply:
Done in 9ca67a0442b2eb1dee63f9b5bed84eda288c9e03